src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -3,17 +3,20 @@ import { useLocation } from 'react-router-dom';
 import { Helmet } from 'react-helmet';
 import { oneOf, string } from 'prop-types';
 import styled from '@emotion/styled';
-import { C_LUNAR_LIGHT } from '#legacy/psammead-styles/src/colours';
 import pathOr from 'ramda/src/pathOr';
 import { ServiceContext } from '#contexts/ServiceContext';
 import { RequestContext } from '#contexts/RequestContext';
 import isLive from '#lib/utilities/isLive';
 import useOperaMiniDetection from '#hooks/useOperaMiniDetection';
+import { C_LUNAR_LIGHT } from '#legacy/psammead-styles/src/colours';
 import getAdsAriaLabel from '../utilities/getAdsAriaLabel';
 import { leaderboardStyles, mpuStyles } from '../utilities/adSlotStyles';
 
+// move colour #E6E8EA in the legacy folder as C_GREY_3 once psammead migration is completed
+const C_GREY_3 = '#E6E8EA';
+
 const AdContainer = styled.section`
-  background-color: ${C_LUNAR_LIGHT};
+  background-color: ${() => (isLive() ? C_LUNAR_LIGHT : C_GREY_3)};
```

Comment:
```suggestion
  background-color: ${isLive() ? C_LUNAR_LIGHT : C_GREY_3};
```

you would only use a function here to get component props

Reply:
Fixed this, thanks for spotting this 👍